modules/push-notifications/src/PushNotifications.ts

```diff
@@ -124,16 +129,41 @@ export default class PushNotifications extends ManagedModule {
 
   async getNotificationTokens(
     call: GetNotificationTokensRequest,
-    callback: GetNotificationTokensResponse
+    callback: GetNotificationTokensResponse,
   ) {
     const userId = call.request.userId;
     let errorMessage: string | null = null;
     const tokenDocuments: any = await models.NotificationToken.getInstance()
       .findMany({ userId })
-      .catch((e: Error) => { errorMessage = e.message });
+      .catch((e: Error) => {
+        errorMessage = e.message;
+      });
     if (errorMessage) {
       return callback({ code: status.INTERNAL, message: errorMessage });
     }
     return callback(null, { tokenDocuments });
   }
+
+  async sendNotification(call: SendNotificationRequest, callback: SendNotificationResponse) {
+    let data = call.request.data;
+    const params: any = {
```

Comment:
the sendTo should be a userId, that identifies the user to which the notification will be sent